src/components/Tabs.vue

```diff
@@ -113,7 +113,7 @@
     },
 
     methods: {
-      handleMouseDown({target, clientY}) {
```

Comment:
This line was changed due to it causing the linter to fail (because target is unused).